revscoring/features/wikibase/tests/alan_touring.json

```diff
@@ -0,0 +1 @@
: "item"}, "type": "wikibase-entityid"}, "property": "P143", "snaktype": "value"}]}, "hash": "2f183743080f8e1970847d13a04fde9a796f1476"}], "id": "Q7251$CFFE701D-1608-434A-894F-1ACE6BFCB763"}], "P463": [{"rank": "normal", "mainsnak": {"hash": "8474eaafda73012ebf544ddc74abada7b467a4e4", "datavalue": {"value": {"numeric-id": 123885, "entity-type": "item"}, "type": "wikibase-entityid"}, "property": "P463", "snaktype": "value"}, "type": "statement", "id": "q7251$DB4F069D-3A9E-4E44-865F-8A4DAE41A2BE"}], "P1207": [{"rank": "normal", "mainsnak": {"hash": "1d9414a674a56fd695c5fbc3bbc175ea9c86e8b6", "datavalue": {"value": "n98045497", "type": "string"}, "property": "P1207", "snaktype": "value"}, "type": "statement", "id": "Q7251$43E371F6-CDA6-4F7C-91C3-8442FDEB5458"}], "P227": [{"rank": "normal", "mainsnak": {"hash": "94d761c62ba3714aa006b25fa286f4b16f993145", "datavalue": {"value": "118802976", "type": "string"}, "property": "P227", "snaktype": "value"}, "type": "statement", "references": [{"snaks-order": ["P143"], "snaks": {"P143": [{"hash": "ed26fcd5570c4e73da1b33301244ec0494789f52", "datavalue": {"value": {"numeric-id": 48183, "entity-type": "item"}, "type": "wikibase-entityid"}, "property": "P143", "snaktype": "value"}]}, "hash": "004ec6fbee857649acdbdbad4f97b2c8571df97b"}, {"snaks-order": ["P143"], "snaks": {"P143": [{"hash": "02a0cb0376dece649d2d09efa2105a8b33b4a1a3", "datavalue": {"value": {"numeric-id": 1419226, "entity-type": "item"}, "type": "wikibase-entityid"}, "property": "P143", "snaktype": "value"}]}, "hash": "9f454a27f5efb737e03ba11bd3e85a1ea1c08a7d"}], "id": "q7251$9AAB8B86-39F4-488C-9273-651C16ED03EE"}], "P646": [{"rank": "normal", "mainsnak": {"hash": "9865f15a1362fc28b2976b8e55cc632dfed40df8", "datavalue": {"value": "/m/0n00", "type": "string"}, "property": "P646", "snaktype": "value"}, "type": "statement", "references": [{"snaks-order": ["P248", "P577"], "snaks": {"P248": [{"hash": "62e345446a0c505d83139055fc6affea7d82887b", "datavalue": {"value": {"numeric-id": 15241312, "entity-type": "item"}, "type": "wikibase-entityid"}, "property": "P248", "snaktype": "value"}], "P577": [{"hash": "326dbe1cf6754fbdae762db3874856de204141f4", "datavalue": {"value": {"before": 0, "timezone": 0, "calendarmodel": "http://www.wikidata.org/entity/Q1985727", "time": "+2013-10-28T00:00:00Z", "after": 0, "precision": 11}, "type": "time"}, "property": "P577", "snaktype": "value"}]}, "hash": "af38848ab5d9d9325cffd93a5ec656cc6ca889ed"}], "id": "Q7251$FBBEEFEF-1BD3-4625-966C-94C44D375964"}], "P1343": [{"qualifiers": {"P854": [{"hash": "3dbfcd60b0730da8cb7240e581a90685b9e3b8df", "datavalue": {"value": "http://www.larousse.fr/encyclopedie/personnage/Alan_Mathison_Turing/147690", "type": "string"}, "property": "P854", "snaktype": "value"}]}, "type": "statement", "id": "Q7251$4D7F2655-DA96-46A1-BAD2-0A9CDB66F94B", "rank": "normal", "qualifiers-order": ["P854"], "mainsnak": {"hash": "3c5601ff703aba852dc449f7fa2e7bd940e53a80", "datavalue": {"value": {"numeric-id": 17
```

Comment:
It's hard to tell what you are referring to, but the test cases pass, so I suspect that the JSON is valid.